src/components/common/Card/Card.tsx

```diff
@@ -1,62 +1,53 @@
 import React from "react";
+import { useNavigate } from "react-router";
 import { ProfileImage, SkillIcon } from "..";
-import { ReactComponent as HeartIcon } from "../../../assets/icons/icon_heart.svg";
-import { ReactComponent as HeartFillIcon } from "../../../assets/icons/icon_heart_fill.svg";
 import { IPost } from "../../../types/model";
+// import LikeBtn from "../LikeBtn";
 import * as S from "./style";
 
-type CardInterface = { post?: IPost } & { user?: string };
+type CardInterface = { post?: IPost } & { userId?: string };
 
-const Card: React.FC<CardInterface> = ({ post = dummy, user = null }) => {
-  const isClickLike = post.likes.some((item) => item.user === user);
+const Card: React.FC<CardInterface> = ({ post, userId = null }) => {
+  const postTitle = post ? JSON.parse(post.title) : dummy;
+  const { _id, author, image, likes } = post;
+  const { title, parts, expectedDate } = postTitle;
+  const navigate = useNavigate();
 
-  const onClickCard = () => {
-    alert("Card");
-    // 관련 상세페이지로 이동
-  };
+  const createAt = new Date(post.createdAt);
+  const Year = createAt.getFullYear();
+  const getMonth = createAt.getMonth() + 1;
+  const getDate = createAt.getDate();
+  const month = getMonth < 10 ? "0" + getMonth : getMonth;
+  const date = getDate < 10 ? "0" + getDate : getDate;
+  const createDate = `${Year}-${month}-${date}`;
```

Comment:
- 컴포넌트 외부 함수로 추출하면 가독성이 더 좋아질 것 같습니다!
- Year만 대문자로 시작하는 것이 마음이 아프군요..
- getMonth, getDate 변수명은 저희가 정한 함수명 컨벤션이라 다른 이름을 사용해야 헷갈리지 않을 것 같습니다

Reply:
이부분은 사용하면 형진님과 같이 쓸것 같으니 한번 형진님과 의논해 보겠습니다~~

Reply:
- 변수명은 소문자 camelCase로 하기로 해서 Year -> year로 쓰는게 적절해보여요 ☺️
- 코드 가독성을 위해 temp성 변수로 빼두신거 같은데 변수 네이밍을 짓기 애매해서 바로 불러와서 사용해도 괜찮을거 같아요
```suggestion
  const createAt = new Date(post.createdAt);
  const year = createAt.getFullYear();
  const month = (createAt.getMonth() + 1) < 10 ? "0" + (createAt.getMonth() + 1 ): (createAt.getMonth() + 1);
  const date = createAt.getDate() < 10 ? "0" + createAt.getDate() : createAt.getDate();
  const createDate = `${year}-${month}-${date}`;
```